Overview.bs

```diff
@@ -189,14 +204,25 @@ Performant implementations should incorporate a similar mechanism which can sele
 likely to be needed in the future and preemptively load them. This will improve performance by reducing
 the need to make additional requests for missing codepoints. The set of codepoints the client has and
 the set they are requesting can be used in conjunction with codepoint occurrence frequency and codepoint
-usage in languages/scripts  to make predictions on which additional codepoints are likely to be needed.
-
-Under range request the prediction mechanism will need to be part of the client implementation. In
-patch subset either the client and/or server implementation can include a prediction mechanism. A side
-benefit to a client side prediction mechanism is providing some obfuscation of the specific codepoints
-required by the client. This is further discussed in [[#content-inference-from-character-set]].
-
-### Deciding between Patch Subset and Range Request ### {#patchsubset-vs-rangerequest}
+usage in languages/scripts to make predictions on which additional codepoints are likely to be needed.
+
+With IFTB the prediction mechanism is a necessary part of the font format conversion, assigning glyphs
+to bins according to language/script and frequency. In patch subset either the client and/or server
+implementation can include a prediction mechanism. A side benefit to a client side prediction mechanism
+is providing some obfuscation of the specific codepoints required by the client. This is further
+discussed in [[#content-inference-from-character-set]].
+
+### Deciding between Patch Subset and IFTB ### {#patchsubset-vs-iftb}
+
+(XXX This whole section needs updating. Patch-subset is likely to return fewer codepoints, although
+with prediction and privacy that becomes less true. IFTB requires no server-side compression and
+static arrangement
+of bins makes it more compatible with caching, including regional caching. ("More compatible" in the
+sense that chunk files will see a higher cache hit rate compared with subset and patch files.) All IFTB
+data is compressed at Brotli level 11 upfront.  IFTB transfers all other tables
```

Comment:
Brotli level 11 is an implementation detail of the current open source brotli encoder, but not actually something part of the brotli standard. Instead maybe just say maximum quality?